sdk/cloudmachine/Azure.Provisioning.CloudMachine/src/FeaturesBuiltIn/AIFoundryFeature.cs

```diff
@@ -69,8 +83,23 @@ protected internal override void EmitConnections(ConnectionCollection connection
         /// <returns>A placeholder or no-op resource, as provisioning is out-of-band for now.</returns>
         protected override ProvisionableResource EmitResources(ProjectInfrastructure cm)
         {
-            // In the future, generate real provisioning resources for Foundry
-            return new NoOpResource();
+            var cmId = cm.Id;
+            AIFoundryCognitiveServiceCdk cs = new($"{cmId}_foundry", $"{cmId}-foundry");
```

Comment:
Do we need a role assignment for these? e.g.

`RequiredSystemRoles.Add(...)`

Reply:
I actually just removed it. It's not needed.